Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -878,7 +878,10 @@ private void ReconcilePointers()
             }
             if (gazePointer != null)
             {
-                gazePointerStateMachine.UpdateState(NumNearPointersActive, NumFarPointersActive);
+                gazePointerStateMachine.UpdateState(
+                    NumNearPointersActive,
+                    NumFarPointersActive,
+                    InputSystem.EyeGazeProvider.IsEyeGazeValid);
```

Comment:
Can this be null ever?

Reply:
InputSystem is a prereq to this file (i.e. other places have the same level of dependency)

EyeGazeProvider is set up by the input system uniquely within its startup path and startup will fail if there is no gaze provider given.

So tl;dr Not as far as I can see.